frontend/src/components/GameEntry.css

```diff
@@ -0,0 +1,60 @@
+.root{
+    --glow-intensity: 0.0
+}
+@keyframes pulsate {
+    0% {
+        box-shadow: 0 0 calc((var(--glow-intensity) * 20px)) rgba(255, 192, 0, calc((var(--glow-intensity) * 0.5)));
+    }
+    50% {
+        box-shadow: 0 0 calc((var(--glow-intensity) * 40px)) rgba(255, 192, 0, calc((var(--glow-intensity) * 1.0)));
+    }
+    100% {
+        box-shadow: 0 0 calc((var(--glow-intensity) * 20px)) rgba(255, 192, 0, calc((var(--glow-intensity) * 0.5)));
+    }
+}
+
+
+.gameCard {
+    /* Add shadows to create the "card" effect */
+    /*box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);*/
+    /*box-shadow: 0 0 200px rgba(255,192,0, var(--glow-intensity));*/
+    border-radius: 10px;
+    /*background-color: #ffffff*/
+    /*transition: 0.3s;*/
```

Comment:
Ist fürs Übungsprojekt vollkommen okay. In einem echten Projekt, sollte aber natürlich kein auskommentierter Code eingecheckt werden.

Reply:
✔️